src/samplers/string.js

```diff
@@ -60,6 +60,10 @@ function uriSample() {
   return 'http://example.com';
 }
 
+function uuidSample(min, max, propertyName) {
```

Comment:
Underscore unused variables

```suggestion
function uuidSample(_min, _max, propertyName) {
```

Reply:
Updated: https://github.com/Redocly/openapi-sampler/pull/106/files#diff-bd2268fad45114c88f3962f59318158fR63